targets/esp-idf/io.c

```diff
@@ -0,0 +1,143 @@
+/* Copyright JS Foundation and other contributors, http://js.foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <stdarg.h>
+#include <stdlib.h>
+#include <string.h>
+#include "esp_log.h"
+
+#include "jerryscript-port.h"
+#include "jerryscript-debugger.h"
+
+static const char TAG[] = "JS";
+
+static esp_log_level_t crosslog(jerry_log_level_t level) {
+	switch(level) {
+	case JERRY_LOG_LEVEL_ERROR: return ESP_LOG_ERROR;
+	case JERRY_LOG_LEVEL_WARNING: return ESP_LOG_WARN;
+	case JERRY_LOG_LEVEL_DEBUG: return ESP_LOG_DEBUG;
+	case JERRY_LOG_LEVEL_TRACE: return	ESP_LOG_VERBOSE;
```

Comment:
There's extra whitespace in the last return statement.

Reply:
corrected